src/input/mpegts/linuxdvb/linuxdvb_ddci.c

```diff
@@ -104,8 +104,17 @@ struct linuxdvb_ddci
   linuxdvb_ddci_wr_thread_t  lddci_wr_thread;
   linuxdvb_ddci_rd_thread_t  lddci_rd_thread;
 
+#if ENABLE_DDCI_MCD
+  /* list of associated services */
+  TAILQ_HEAD(,service)       lddci_services;
```

Comment:
Use idnode_set here so we can drop the new member from service_t structure (see src/idnode.h).

Reply:
I will have a look on that.